src/components/Graph/GraphOptions.js

```diff
@@ -1,4 +1,29 @@
-﻿export default function graphOptions(configuration) {
+﻿const commentDataPointColour = 'rgb(25,25,112, 0.8)'
+const dataPointColour = 'rgb(30, 144, 255, 0.5)'
+const graphColour = 'rgb(30, 144, 255)'
+
+const setBackgroundColor = (ctx) => {
+  if (ctx.raw && ctx.raw.comment) {
+    return commentDataPointColour
+  }
+  return dataPointColour
+}
+
+// datasets for graph
+export function getGraphData(data) {
+  return {
+    datasets: [
+      {
+        data: data,
+        borderColor: graphColour,
```

Comment:
small comment on different of using English and British English for color & colour

Reply:
Changed to color, as GraphJS uses American English spelling